docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -359,12 +359,17 @@ Remote environment variables are available via facts in the 'ansible_env' variab
 How do I generate crypted passwords for the user module?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-The mkpasswd utility that is available on most Linux systems is a great option:
+Ansible ad-hoc command is the easiest option:
+
+.. code-block:: shell-session
+
+    ansible all -i localhost, -m debug -a "msg={{ 'mypassword' | password_hash('sha512', 'mysecretsalt') }}"
```

Comment:
Instead of 

```
ansible all -i localhost, -m
```

We should use 

```
ansible localhost -m ...
```

This doesn't require an inventory and relies on implicit localhost

Reply:
That's true but it prints several warnings which might scare some users that they do something wrong. I would prefer to use the original version which doesn't print any warnings.

Reply:
@jtyr what are the warnings? Can you post them in a comment?